integration-tests/_tests/12-auth-create-user.test.ftd

```diff
@@ -48,7 +48,7 @@ fastn.assert.not_empty(fastn.http_response.errors.payload);
 -- fastn.post: login with invalid user shouldn't work
 url: /-/auth/login/?provider=emailpassword
 body: {"username": "someone", "password": "veryrealpasswd"}
-http-status: 400
+http-status: 200
```

Comment:
This should be default, so let's leave this line out. (do check if fastn.test def asserts http code 200 even if the http-status is not explicitly passed).